bin/runtime/src/lib.rs

```diff
@@ -426,19 +434,36 @@ impl pallet_staking::WeightInfo for PayoutStakersDecreasedWeightInfo {
             Weight
         ),
         (get_npos_targets(v: u32), SubstrateStakingWeights, Weight),
-        (set_staking_limits(), SubstrateStakingWeights, Weight),
         (chill_other(), SubstrateStakingWeights, Weight)
     );
+
+    fn set_staking_configs_all_set() -> Weight {
+        todo!()
```

Comment:
It's fine to use the default here: `(set_staking_configs_all_set, SubstrateStakingWeights, Weight),` in the above `wrap_methods` macro.